### PR TITLE
UW-566 Support FV3 global-domain configuration

### PR DESCRIPTION
**Pull request by maddenp-noaa**
**Synopsis**

Make lateral boundary conditions required only when running FV3 with a regional domain.

**Type**

- [x] Bug fix (corrects a known issue)
- [x] Documentation
- [x] Enhancement (adds a new functionality)

**Impact**

- [x] This is a non-breaking change (existing functionality continues to work as expected)

**Checklist**

- [x] I have added myself and any co-authors to the PR's _Assignees_ list.
- [x] I have reviewed the documentation and have made any updates necessitated by this change.

### CODE REVIEW

File: src/uwtools/tests/test_schemas.py [hunk 1]
```diff
@@ -61,18 +61,18 @@ def ungrib_prop():
 
 
 def test_schema_chgres_cube():
-    d = {
+    config = {
```
Comment (maddenp-noaa) [inline src/uwtools/tests/test_schemas.py:64]: Perhaps a slightly more meaningful name.

Comment (NaureenBharwaniNOAA) [reply src/uwtools/tests/test_schemas.py:64]: This is a great change.

**Review by NaureenBharwaniNOAA — APPROVED** [review]
Looks great! I like the name change from `d` to `config`.

**Review by elcarpenterNOAA — APPROVED** [review]
LGTM!

**Review by WeirAE — APPROVED** [review]
LGTM!

**Review by christinaholtNOAA — APPROVED** [review]
LGTM.

**Pull request merged**